packages/react-dom/src/events/__tests__/SelectEventPlugin-test.internal.js

```diff
@@ -58,34 +58,36 @@ describe('SelectEventPlugin', () => {
     expect(mouseup).toBe(null);
   });
 
-  it('should extract if an `onSelect` listener is present', () => {
-    class WithSelect extends React.Component {
-      render() {
-        return <input type="text" onSelect={this.props.onSelect} />;
-      }
-    }
-
-    var cb = jest.fn();
-
-    var rendered = ReactTestUtils.renderIntoDocument(
-      <WithSelect onSelect={cb} />,
+  it('should register event only if the `onSelect` listener is present', () => {
+    var select = jest.fn();
+    var onSelect = event => {
+      expect(typeof event).toBe('object');
+      expect(event.type).toBe('select');
+      expect(event.target).toBe(node);
+      select(event.currentTarget);
+    };
+
+    var childContainer = document.createElement('div');
+    var childNode = ReactDOM.render(
+      <input type="text" onSelect={onSelect} />,
+      childContainer,
     );
-    var node = ReactDOM.findDOMNode(rendered);
+    document.body.appendChild(childContainer);
 
-    node.selectionStart = 0;
-    node.selectionEnd = 0;
+    var node = ReactDOM.findDOMNode(childNode);
```

Comment:
There's no need to `findDOMNode`. `render()` already gave you the right node. Old test just wasn't updated to reflect this.